egs/wsj/asr1/run.sh

```diff
@@ -134,6 +137,19 @@ if [ ${stage} -le 2 ] && [ ${stop_stage} -ge 2 ]; then
         data2json.sh --feat ${feat_recog_dir}/feats.scp \
             --nlsyms ${nlsyms} data/${rtask} ${dict} > ${feat_recog_dir}/data.json
     done
+
+    ### Filter out short samples which lead to `loss_ctc=inf` during training
+    ###  with the specified configuration.
+    # `subsample` is in ${train_config}
+    min_io_ratio=$(grep -P 'subsample' ${train_config} | grep -Po '[\d_]+' | \
```

Comment:
This is risky because subsample is only available when we use BLSTMP. It is disabled for BLSTM and VGGBLSTM*

Reply:
Is there a simple way to get around this problem? I am not very sure how the subsampling option will be specified in different cases.
Or do we have to specify the parameter `min_io_ratio` manually?

Reply:
how about copying the code from   https://github.com/espnet/espnet/blob/212e5c54d2be9775a55199c278cd830f745f5317/espnet/nets/pytorch_backend/e2e_asr_mix.py#L178 and get the actual subsampling?
but this is a bit risky and the perfect solution is to make (an executable function) of the above part, import this for all subsampling parts (asr, st, etc.), and call this function inside this function.
thus we can get the actual subsampling from the training config file.